netmaster/gstate/gstate.go

```diff
@@ -436,3 +488,40 @@ func (gc *Cfg) UnassignNetwork(networkName string) error {
 
 	return nil
 }
+
+//CheckInBitRange to check if range includes inuse vlans
+func (gc *Cfg) CheckInBitRange(ranges, inUse, pktTagType string) bool {
+
+	tags := strings.Split(inUse, ",")
+
+	if len(inUse) == 0 {
+		return true
+	}
+	minUsed := 0
+	maxUsed := 0
+	if strings.Contains(tags[0], "-") {
+		minUsed, _ = strconv.Atoi(strings.Split(tags[0], "-")[0])
+		maxUsed, _ = strconv.Atoi(strings.Split(tags[0], "-")[1])
+	} else {
+		minUsed, _ = strconv.Atoi(tags[0])
+		maxUsed = minUsed
+	}
+
+	if len(inUse) > 1 {
+		if strings.Contains(tags[len(tags)-1], "-") {
+			maxUsed, _ = strconv.Atoi(strings.Split(tags[len(tags)-1], "-")[1])
+		} else {
+			maxUsed, _ = strconv.Atoi(strings.TrimSpace(tags[len(tags)-1]))
+		}
+	}
+
+	r, err := netutils.ParseTagRanges(ranges, pktTagType)
+	if err != nil {
+		return false
+	}
+
+	if r[0].Min > minUsed || r[0].Max < maxUsed {
```

Comment:
We dont support disjoint ranges for vxlan at the moment.  Unless there is a need due the amount of vxlan range space available. We can enhance it should need be